### PR TITLE
Redesign the school dashboard

### PR DESCRIPTION
**Pull request by leoapost**
### Trello card
https://trello.com/c/Bm7HqHUi

### Context
Dashboard redesign 

### Changes proposed in this pull request
- Update the dashboard view and partial views
- Update the feature tests

### Guidance to review
|Before|After|
|-|-|
| before | after|

### CODE REVIEW

File: app/views/schools/dashboards/_help_and_support.html.erb [hunk 1]
```diff
@@ -0,0 +1,22 @@
+<div class="<%= @current_school.private_beta? ? 'govuk-grid-column-one-half' : 'govuk-grid-column-full' %>">
+  <div id="help-and-support" class="panel govuk-!-margin-bottom-5">
```
Comment (jkempster34) [inline app/views/schools/dashboards/_help_and_support.html.erb:2]: flex: 1

**Comment by jkempster34** [comment]
> Had a poke around and everything looks nice 👍 .
> 
> Could we apply `display: flex` and `flex: 1` to make the boxes the same height here? image

Actually, this breaks the responsive behaviour. Not sure if its worth it 
image

**Comment by leoapost** [comment]
> Had a poke around and everything looks nice +1 .
> 
> Could we apply `display: flex` and `flex: 1` to make the boxes the same height here? image

We tried and in some cases the white space was huge. Didn't look that great, so we decided to not make them the same height.

**Review by jkempster34 — APPROVED** [review]

**Comment by MylesJarvis** [comment]
We can also 🚢  this one in the morning. I'll do a final review then.

**Pull request merged**